packages/helpers/src/media.js

```diff
@@ -0,0 +1,53 @@
+import tokens from '@gympass/yoga-tokens';
+import { css } from 'styled-components';
+
+import hide from './hide';
+
+const { breakpoints } = tokens;
+
+const media = {
+  not: {},
+};
+
+export const not = isNot => (isNot ? 'not all and ' : '');
+
+const getRange = (width, range) => {
+  if (Array.isArray(width)) {
+    const [min, max] = width;
+
+    const {
+      [min]: { width: minBreakpoint },
+      [max]: { width: maxBreakpoint },
+    } = breakpoints;
+
+    return `(min-width: ${minBreakpoint}px) and (max-width: ${maxBreakpoint}px)`;
+  }
+
+  return `(${range}-width: ${breakpoints[width].width}px)`;
+};
+
+export const matcher = (width, isNot = false, range = 'min') => (
+  ...style
+) => css`
+  @media ${not(isNot)}${getRange(width, range)} {
+    ${css(...style)}
+  }
+`;
+
+Object.keys(breakpoints).reduce((acc, label) => {
+  acc[label] = matcher(label);
+  acc.not[label] = matcher(label, true);
+
+  return acc;
+}, media);
```

Comment:
This is not being used, right? 🤔

Reply:
This is the code responsible for adding the defaults: 
 `media.xxs`, `media.xs`, `media.sm`, `media.md`, `media.lg`, `media.xl`, `media.xxl`, `media.xxxl`, `media.not.xxs`, `media.not.xs`, `media.not.sm`, `media.not.md`, `media.not.lg`, `media.not.xl`, `media.not.xxl`, and `media.not.xxxl`

Reply:
Hmmm, got it! I did not see the `media` as the initial value. 🤦🏼 

Maybe we could use another approach to make this more "obvious":

```suggestion
// Add default values.
Object.keys(breakpoints).forEach((label) => {
  media[label] = matcher(label);
  media.not[label] = matcher(label, true);
});
```